src/DetectChanges/BCBreak/FunctionBased/ParameterTypeChanged.php

```diff
@@ -79,7 +79,6 @@ private function typeToString(?ReflectionType $type): string
             return 'no type';
         }
 
-        return ($type->allowsNull() ? '?' : '')
-            . $type->__toString();
+        return $type->__toString();
```

Comment:
`ReflectionType#__toString()` now includes nullability as per `roave/better-reflection:5.0.0` - this is a BC break that led to some simplification and also code changes all around the place.